### PR TITLE
chore: chore_release-8.0.0->edge for alpha 5

### PR DESCRIPTION
**Pull request by sfoster1**
Significant changes for this one, please review.

### CODE REVIEW

File: app/src/resources/runs/useNotifyRunQuery.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ import type { Run, HostConfig } from '@opentrons/api-client'
 import type { QueryOptionsWithPolling } from '../useNotifyDataReady'
 import type { NotifyTopic } from '../../redux/shell/types'
 
+// TODO(jh, 08-21-24): Abstract harder.
```
Comment (mjhuff) [inline app/src/resources/runs/useNotifyRunQuery.ts]:
```suggestion
```
The abstracting harder is done.

File: app/src/resources/useNotifyDataReady.ts [hunk 2]
```diff
@@ -34,6 +34,8 @@ interface UseNotifyDataReadyResults<TData, TError> {
   queryOptionsNotify: QueryOptionsWithPolling<TData, TError>
   /* Whether notifications indicate the server has new data ready. Always returns false if notifications are disabled. */
   shouldRefetch: boolean
+  /* Whether notifications are enabled as determined by client options and notification health. */
+  isNotifyEnabled: boolean
```
Comment (mjhuff) [inline app/src/resources/useNotifyDataReady.ts]:
```suggestion
```
We fortunately don't need this anymore.

File: robot-server/robot_server/runs/run_data_manager.py [hunk 3]
```diff
@@ -357,7 +357,7 @@ async def update(self, run_id: str, current: Optional[bool]) -> Union[Run, BadRu
             parameters = self._run_orchestrator_store.get_run_time_parameters()
             run_resource = self._run_store.get(run_id=run_id)
 
-        self._runs_publisher.publish_runs_advise_refetch(run_id)
+        await self._runs_publisher.publish_runs_advise_refetch_async(run_id)
```
Comment (mjhuff) [inline robot-server/robot_server/runs/run_data_manager.py]:
```suggestion
        self._runs_publisher.publish_runs_advise_refetch_(run_id)
```
Are we sure on this one? I thought all the actual publishing is done synchronously now.

File: robot-server/robot_server/service/notifications/publishers/runs_publisher.py
Comment (mjhuff) [inline robot-server/robot_server/service/notifications/publishers/runs_publisher.py:1]: Yeah, we want all the sync logic I believe. We no longer do async publishing.

**Review by mjhuff — COMMENTED** [review]
Some comments, mainly with the server notification changes. Thanks for doing this!

File: robot-server/robot_server/service/notifications/publishers/runs_publisher.py
Comment (sfoster1) [reply robot-server/robot_server/service/notifications/publishers/runs_publisher.py:1]: oh I went the wrong way on those

**Review by smb2268 — APPROVED** [review]
Everything I contributed to (quick transfer, pipette wizard flows, run summary and confirm cancel routing) looks good to me! I also looked over the other JS changes and don't see any red flags.

**Review by CaseyBatten — APPROVED** [review]
Changes for thermocycler handling on app and pipette press distance values all check out, quick transfer details seem in line with recent changes.

**Pull request merged**